core/src/main/scala/org/apache/spark/deploy/worker/DriverWrapper.scala

```diff
@@ -66,4 +70,16 @@ object DriverWrapper {
         System.exit(-1)
     }
   }
+
+  private def setupDependencies(loader: MutableURLClassLoader, userJar: String): Unit = {
+    var jars = sys.props.get("spark.jars").orNull
+    val resolvedMavenCoordinates = DependencyUtils.resolveMavenDepndencies()
+
+    if (!StringUtils.isBlank(resolvedMavenCoordinates)) {
+      jars = SparkSubmit.mergeFileLists(jars, resolvedMavenCoordinates)
+    }
+
+    jars = DependencyUtils.resolveAndDownloadJars(jars, userJar)
```

Comment:
Instead of using a `var` I find it cleaner to define a different variable (e.g. `val localJars = DependencyUtils...`).